Different colors to override stroke and fill

tovgroliv opened the pull request:
I really think that for modern UI it is necessary to separate the fill color and the stroke color. I hope this pull request is needed. Thanks for your work.

paulushub commented:
@tovgroliv Thanks for the contribution. 

> I really think that for modern UI it is necessary to separate the fill color and the stroke color. 

Most of such color change uses are monochrome. Also, there is a support for `CustomBrushes`, 
so I do not know how these options are used in practice.

Now, please leave the current property `OverrideColor` there, removing it will break existing codes.
The current color property `OverrideColor` and the fill property `OverrideFill` are both nullable, so you can check which is set
and apply it accordingly.
And for the demo, you can add another control instance to the list and implement your new features.
This is how the source is implemented. The original implementation is a string `Source`, which expected a relative URI.
I added the `UriSource`, which of the type `Uri`.

Finally, we are adding documentation support so please document the added properties, and it will help others if you add an article to show how to use the added properties.

tovgroliv commented:
Thanks for the answer.

I finished. Now OverrideColor is leaved. I added OverRideStrokeColor, which replaces the stroke color if necessary.
I also added an application example and a test to Example. It is located next to OverrideColor.

This is a very small change that does not require detailed documentation.

This behavior can be useful, for example, to create an animation for installing a favorite item. For a non-favorite element, a star without fill is set, and for a favorite element, a star with fill is set. This PullRequest allows you to do this much faster. I hope not only for me (:

jogibear9988 commented on Source/SVGImage/SVG/SvgIconBase.cs:
```diff
@@ -46,6 +46,7 @@ protected SvgIconBase()
         #region Public Properties        
 
         public Color? OverrideColor { get; set; }
+        public Color? OverrideBrush { get; set; }
```
Why is it here Named: OverrideBrush?

jogibear9988 commented:
the rest LGTM

jogibear9988 commented:
LGTM

paulushub commented:
> This is a very small change that does not require detailed documentation.

Add XML documentations to the added properties.

paulushub commented:
@tovgroliv Sorry, closing this pull request for now. 
Anytime you are ready, please resubmit it to the newly created **develop** branch, which is now the `default`.
Releases will be made from the `master` branch.
